README.md

```diff
@@ -131,7 +134,19 @@ Let's say the Posts in the previous question are available when you visit `http:
   3. If Step 1 is unsuccessful, print an error message to the console
 
 ```js
-// Your answer goes here...
+var post = $.get("post.rb", function(){
```

Comment:
You can't make a request to a ruby file. Instead use the base URL (plus whatever else needs to be added) provided in the question.